packages/support/resources/views/components/modal/index.blade.php

```diff
@@ -54,6 +54,7 @@
     aria-modal="true"
     class="filament-modal {{ $displayClasses }}"
     wire:ignore.self
+    wire:key="{{ $this->id }}-{{ $id }}-modal-container"
```

Comment:
@danharrin don't we use `.` instead of `-` in other places?

Reply:
yeah i fixed it the minute after i merged lol

Reply:
Lol, didn't see that commit. Thanks.